Daytime-Operations/Auxiliary-Telescope/AT-Calibrations/prepare-atcs-for-flat-fields-procedure.rst

```diff
@@ -8,7 +8,54 @@
 Prepare ATCS For Flat Fields Procedure
 ######################################
 
-Under construction.
+Overview
+========
```

Comment:
I believe the content is fine but the page would greatly benefit from the standard structure: 
- Overview 
- Prerequisites (What needs to happen before this procedure is executed)
- Post-condition (What happens next, what's the result of the procedure) 
- Procedure steps per se. 

Some examples are https://obs-ops.lsst.io/Daytime-Operations/Auxiliary-Telescope/AT-Calibrations/LATISS-Combined-Calibrations-Procedure/latiss-combined-calibrations-procedure.html or the more simple https://obs-ops.lsst.io/Nighttime-Operations/Auxiliary-Telescope/AT-On-Sky/Focus-center-absorbPointingOffsets.html

Reply:
I agree with Ioanna. This documentation fits very well on the standard structure (making life even easy!).